process/graduation_criteria.md

```diff
@@ -0,0 +1,115 @@
+# Azure Container Upstream Graduation Criteria v0.1
+The Azure Container Upstream Graduation Criteria owes much of its basis on two existing documents,
+[CNCF Graduation Criteria](https://github.com/cncf/toc/blob/fc161c13a86ba022277a906a10ff51a568406f7c/process/graduation_criteria.adoc) 
+and [.NET Foundation Project Maturity Profile](https://github.com/dotnet-foundation/project-maturity-model/blob/65fdfa43d7e68845bd65638c378cf834d7158b90/maturity-profiles.md). The goal of this document is to describe the process and 
+minimum bar for each increasing level of project maturity to both provide a
+goal for the project maintainers, but also to set expectations for consumers of these
+projects.
+
+## Process
+
+Each project in Container Upstream (AzCU) has an associated maturity level. A proposed AzCU project should enter at the 
+project's preferred maturity level. If the project originates within the AzCU group, then it should begin at the Sandbox
+Stage. If a joining project originates outside of the AzCU group, the preferred maturity level of the project must meet 
+a 2/3 supermajority vote by the AzCU technical oversight committee (TOC). If the vote tally can not be met at the preferred 
```

Comment:
is the AzCU technical oversight committee TBD?

Reply:
Yes. I think something akin to the CNCF model of TOC is needed for review / oversight, but I've intentionally not defined it outside of it's role.